### DISCUSSION TITLE
Feature Request : Instagram Import

### DISCUSSION THREAD

**Discussion by PeacheyMcKeitch** [mastodon/mastodon#28489] [General]
Are there any plans to add an instagram import to mastodon? Like micro.blog or pixelfed have.

Want to merge all my previous social media into one thing basically (Ie use it for both a twitter and instagram replacement.) But its a shame to loose all my previous stuff.

Currently doing this fine with micro.blog which is generally fantastic but unfortunately the (well intentioned) lack of being able to see who's following you on micro.blog is turning out to be a bit of a deal breaker for me.

Cheers!

**Comment by ytilotti** [comment]
Hi @PeacheyMcKeitch, have you more informations on the subject. Personnaly I want to import data from Facebook, Instagram or Pixelfed in Mastodon, but I found nothing.
